composer.json

```diff
@@ -18,7 +18,7 @@
     "ext-json": "*",
     "ext-openssl": "*",
     "google/apiclient": "^2.0",
-    "google/apiclient-services": "~0.1",
+    "google/apiclient-services": "dev-master",
```

Comment:
This is probably okay to change since this `composer.json` is more of a dev-dependency (you can't install a fully-working version of the plugin by requiring `google/google-site-kit`).

It's a bit odd that the new version hasn't been tagged yet but 🤷‍♂️ 

I'm okay with this change if @felixarntz is (we control when its updated anyways) but I would prefer we used a version constraint instead; I understand the latest tag doesn't include `dataState` yet.

Reply:
Good catch - I hadn't noticed that the change hasn't been released yet. Let's roll this change here back and wait for the next release instead. I've just emailed the maintainer of the PHP library and will keep an eye out for the update. We can have this issue wait in MR until then.